sensu-plugins-passenger.gemspec

```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |s| # rubocop:disable Metrics/BlockLength
   s.test_files             = s.files.grep(%r{^(test|spec|features)/})
   s.version                = SensuPluginsPassenger::Version::VER_STRING
 
-  s.add_runtime_dependency 'nokogiri',     '~> 1.8', '>= 1.8.2'
+  s.add_runtime_dependency 'nokogiri',     '~> 1.10.0'
```

Comment:
Can you pin to 1.11? There’s a vulnerability in 1.10

Reply:
But they dropped support for 2.3 & 2.4 ruby version

Reply:
Shall we use 1.8.X?

Reply:
Use the latest version that will work then

Reply:
Update PR to use version 1.8.5